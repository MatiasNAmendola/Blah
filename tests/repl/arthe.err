^<tokenize>
^<parse>
^<parse>
^<parse>
^3$
^<parse>
^<eval>
