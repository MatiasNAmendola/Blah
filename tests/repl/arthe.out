<tokenize> umatched token '~'
<parse> [Dk MULT] [INT 5]
<parse> [Ds (So (Oa (Ac (Ca (At (Tf (Fn (Ni 5)))))))),Dk MINUS,Dk PLUS,Da (At (Tf (Fp (Ni 3))))] []
<parse> [Dk MINUS] []
3
<parse> [Ds (So (Oa (Ac (Ca (At (Tf (Fp (Ni 5))))))))] [RPAREN]
<eval> divide by zero
